cost-model/src/cost_model.rs

```diff
@@ -396,13 +394,8 @@ mod tests {
             &token_transaction,
             &FeatureSet::all_enabled(),
         );
-        assert_eq!(
```

Comment:
This seems like a breaking change, since now the total cost/sum is different - is this just a difference in cost or will it actually lead to consensus mismatch?

Let's say I request 12000 CUs via compute budget, and have 1 custom ix that uses 12000 CUs. Previously transaction cost would have been 12k (bpf) + 300 (builtin) + (other). Would my transaction now fail on exceeded limit, since I use 300 for cost model + 12k on my ix which is higher than my budget of 12k?

Reply:
That transaction will fail by SVM at `invoke_context.consume_checked(cu_to_consumme)` since that feature gate enabling. It is consensus breaking, but not by this pr.